lib/v2/mihoyo/maintainer.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+    '/bbs/img-ranking/:game/:forum_type?/:cate_type?/:ranking_type?/:page_size?/:last_id?': ['CaoMeiYouRen'],
```

Comment:
Consider using query string style like Twitter as there're quite a few parameters.

Regarding `page_size`, you consider obtaining the common parameter `ctx.query.limit`